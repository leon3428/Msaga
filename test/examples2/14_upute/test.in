<prijevodna_jedinica>
 <prijevodna_jedinica>
  <prijevodna_jedinica>
   <prijevodna_jedinica>
    <vanjska_deklaracija>
     <deklaracija>
      <ime_tipa>
       <specifikator_tipa>
        KR_INT 1 int
      <lista_init_deklaratora>
       <init_deklarator>
        <izravni_deklarator>
         IDN 1 f
         L_ZAGRADA 1 (
         KR_VOID 1 void
         D_ZAGRADA 1 )
      TOCKAZAREZ 1 ;
   <vanjska_deklaracija>
    <definicija_funkcije>
     <ime_tipa>
      <specifikator_tipa>
       KR_INT 3 int
     IDN 3 main
     L_ZAGRADA 3 (
     KR_VOID 3 void
     D_ZAGRADA 3 )
     <slozena_naredba>
      L_VIT_ZAGRADA 3 {
      <lista_naredbi>
       <naredba>
        <naredba_skoka>
         KR_RETURN 4 return
         <izraz>
          <izraz_pridruzivanja>
           <log_ili_izraz>
            <log_i_izraz>
             <bin_ili_izraz>
              <bin_xili_izraz>
               <bin_i_izraz>
                <jednakosni_izraz>
                 <odnosni_izraz>
                  <aditivni_izraz>
                   <multiplikativni_izraz>
                    <cast_izraz>
                     <unarni_izraz>
                      <postfiks_izraz>
                       <postfiks_izraz>
                        <primarni_izraz>
                         IDN 4 f
                       L_ZAGRADA 4 (
                       D_ZAGRADA 4 )
         TOCKAZAREZ 4 ;
      D_VIT_ZAGRADA 5 }
  <vanjska_deklaracija>
   <definicija_funkcije>
    <ime_tipa>
     <specifikator_tipa>
      KR_INT 7 int
    IDN 7 f
    L_ZAGRADA 7 (
    KR_VOID 7 void
    D_ZAGRADA 7 )
    <slozena_naredba>
     L_VIT_ZAGRADA 7 {
     <lista_naredbi>
      <naredba>
       <naredba_skoka>
        KR_RETURN 8 return
        <izraz>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <primarni_izraz>
                       BROJ 8 0
        TOCKAZAREZ 8 ;
     D_VIT_ZAGRADA 9 }
 <vanjska_deklaracija>
  <definicija_funkcije>
   <ime_tipa>
    <specifikator_tipa>
     KR_INT 11 int
   IDN 11 f
   L_ZAGRADA 11 (
   KR_VOID 11 void
   D_ZAGRADA 11 )
   <slozena_naredba>
    L_VIT_ZAGRADA 11 {
    <lista_naredbi>
     <naredba>
      <naredba_skoka>
       KR_RETURN 12 return
       <izraz>
        <izraz_pridruzivanja>
         <log_ili_izraz>
          <log_i_izraz>
           <bin_ili_izraz>
            <bin_xili_izraz>
             <bin_i_izraz>
              <jednakosni_izraz>
               <odnosni_izraz>
                <aditivni_izraz>
                 <multiplikativni_izraz>
                  <cast_izraz>
                   <unarni_izraz>
                    <postfiks_izraz>
                     <primarni_izraz>
                      BROJ 12 1
       TOCKAZAREZ 12 ;
    D_VIT_ZAGRADA 13 }
