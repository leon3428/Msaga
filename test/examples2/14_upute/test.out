<definicija_funkcije> ::= <ime_tipa> IDN(11,f) L_ZAGRADA(11,() KR_VOID(11,void) D_ZAGRADA(11,)) <slozena_naredba>
